client/me/account/main.jsx

```diff
@@ -60,6 +60,7 @@ const Account = localize(
 	React.createClass( {
 		displayName: 'Account',
 
+		// form-base mixin is needed for getDisabledState() (and possibly other uses?)
```

Comment:
It would be awesome to replace `form-base` with Redux Form soon ;)

Reply:
Indeed, but I didn't want to go down the 🐰 :hole: for the sake of the React 16 upgrade 😄